msal/src/telemetry/java/com/microsoft/identity/client/ApiEvent.java

```diff
@@ -44,7 +44,9 @@ private ApiEvent(Builder builder) {
         if (null != builder.mRequestId) {
             setProperty(EventProperty.REQUEST_ID, builder.mRequestId);
         }
-        setProperty(EventProperty.AUTHORITY_NAME, HttpEvent.sanitizeUrlForTelemetry(builder.mAuthority));
+        if (!MsalUtils.isEmpty(builder.mAuthority)) {
```

Comment:
hm, but publicClientApplciation is always created with an authority. If it's not provided, we always use the default one; developer can also provide it thru metadata or constructor.